server/src/main/scala/org/scassandra/server/priming/cors/CorsSupport.scala

```diff
@@ -0,0 +1,43 @@
+package org.scassandra.server.priming.cors
+
+import com.typesafe.scalalogging.LazyLogging
+import org.scassandra.server.ScassandraConfig
+import spray.http.HttpHeaders._
+import spray.http.HttpMethods._
+import spray.http.{AllOrigins, HttpMethods, HttpResponse}
+import spray.routing._
+
+/**
+  * A mixin to provide support for providing CORS headers as appropriate
+  */
+trait CorsSupport {
+  this: HttpService with LazyLogging =>
+
+  private val allowOriginHeader = `Access-Control-Allow-Origin`(AllOrigins)
```

Comment:
It would be a nice eventual enhancement to be able to restrict the origins instead of allowing everything.   That would require adding another config option to `ScassandraConfig`.   Don't think that is needed immediately, but would be nice to eventually have.